docs/templating/markdown.html

```diff
@@ -147,16 +178,26 @@ <h2 class="heading-2"><span class="blue-text bold">Step 2:</span> Create the tem
           <pre><code>swift package generate-xcodeproj</code></pre>
```

Comment:
Since the Views directory isn't code it isn't really required to regenerate the Xcode project. If you make it in xcode then it will be there. If you make it in terminal we can add a line:
```
open -a Xcode.app book.md
```
With a comment about any text editor being able to be used. These comments also applies to the Stencil guide.